<launch>
    <param name="hokuyo_node/port" value="/dev/ttyACM0"/>
    <node pkg="rqt_gui" name="rqt_gui" type="rqt_gui"/>
    <node pkg="joy" name="joy_node" type="joy_node"/>
    <node pkg="hokuyo_node" name="lidar" type="hokuyo_node"/>
    <node pkg="fix" name="nmea_navsat_driver" type="nmea_serial_driver"/> 
    
    <group ns="buzzmobile">
        <rosparam command="load" file="$(find buzzmobile)/constants.yaml"/>
        
        <node pkg="buzzmobile" name="destination"     type="inputer.py"/>
        <node pkg="buzzmobile" name="polyline"        type="maps_querier.py"/>
        <node pkg="buzzmobile" name="route_map"       type="route_mapper.py"/>
        <node pkg="buzzmobile" name="gps_model"       type="gps_mapper.py"/>
        <node pkg="buzzmobile" name="bearing"         type="bearing.py"/>
        <node pkg="buzzmobile" name="world_model"     type="frame_merger.py"/>
        <node pkg="buzzmobile" name="tentacle_frame"  type="steering.py"/>
        <node pkg="buzzmobile" name="steering"   type="steering.py"/>
        <node pkg="buzzmobile" name="lidar_model"     type="lidar_to_frame.py"/>
        <node pkg="buzzmobile" name="car_pose"        type="car_pose_mux.py"/>
        <node pkg="buzzmobile" name="manual_car_pose" type="controller"/>
        <node pkg="buzzmobile" name="car_state"       type="controller"/>
    </group>
</launch>

